--- cmd/wails/internal/template/BUILD.bazel
+++ cmd/wails/internal/template/BUILD.bazel
@@ -2,7 +2,8 @@
 
 go_library(
     name = "template",
     srcs = ["template.go"],
+    embedsrcs = glob(["base/**"]),
     importpath = "github.com/wailsapp/wails/v2/cmd/wails/internal/template",
     importpath_aliases = ["github.com/wailsapp/wails/cmd/wails/internal/template"],
     visibility = ["//cmd/wails:__subpackages__"],
